# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse/Cargo.toml
# Commit: 9493029c76b9
# Full Hash: 9493029c76b93861ca10dbfb29d4178773144416
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 7d5cc211e3f5 -r 9493029c76b9 third_party/rust/mp4parse/Cargo.toml
--- a/third_party/rust/mp4parse/Cargo.toml	Fri Jan 27 19:54:29 2023 +0000
+++ b/third_party/rust/mp4parse/Cargo.toml	Fri Jan 27 19:54:29 2023 +0000
@@ -1,6 +1,6 @@
 [package]
 name = "mp4parse"
-version = "0.13.0"
+version = "0.16.0"
 authors = [
   "Ralph Giles <giles@mozilla.com>",
   "Matthew Gregan <kinetik@flim.org>",
@@ -13,7 +13,7 @@
 documentation = "https://docs.rs/mp4parse/"
 license = "MPL-2.0"
 categories = ["multimedia::video"]
-
+edition = "2018"
 repository = "https://github.com/mozilla/mp4parse-rust"
 
 # Avoid complaints about trying to package test files.
@@ -28,7 +28,6 @@
 [dependencies]
 byteorder = "1.2.1"
 bitreader = { version = "0.3.2" }
-env_logger = "0.8"
 fallible_collections = { version = "0.4", features = ["std_io"] }
 num-traits = "0.2.14"
 log = "0.4"